cyhy_report/customer/generate_report.py

```diff
@@ -67,61 +68,67 @@
 SECTION = None
 SNAPSHOT_HISTORY_LIMIT = 30
 MAX_REPORTCARD_BOX_DISPLAY = 5000
-CRITICAL_AGE_OVER_TIME_CUTOFF_DAYS = 365    # Determines how far back we look at tickets for critical-vuln-ages-over-time figure
-TICKET_AGE_BUCKET_CUTOFF_DAYS = 30  # Dividing line between 'young' and 'old' tickets in Cyber Exposure-esque graphs
-ACTIVE_CRITICAL_AGE_CUTOFF_DAYS = 180 # Max number of days to display in __figure_active_critical_vuln_age_distribution()
-ACTIVE_CRITICAL_AGE_BUCKETS = [(0,7), (7,14), (14,21), (21,30), (30,90)]
+CRITICAL_AGE_OVER_TIME_CUTOFF_DAYS = 365  # Determines how far back we look at tickets for critical-vuln-ages-over-time figure
+TICKET_AGE_BUCKET_CUTOFF_DAYS = (
+    30  # Dividing line between 'young' and 'old' tickets in Cyber Exposure-esque graphs
+)
```

Comment:
Should we kick the comments to the preceding line to improve readability?

Reply:
Good idea- see https://github.com/cisagov/cyhy-reports/pull/55/commits/d6863789f917981fbebbd3c859086f39fe57cf0b.